lib/eunit/src/eunit.erl

```diff
@@ -265,3 +266,49 @@ all_options(Opts) ->
     catch
 	_:_ -> Opts
     end.
+
+%% Delete duplicates of tests
+
+delete_duplicates([_|_] = Tests) ->
+    Candidates = [atom_to_list(T) || T <- Tests, is_atom(T), is_mod_exist(T)],
+    delete_duplicates(Candidates, Tests);
+delete_duplicates(Tests) ->
+    Tests.
+
+is_mod_exist(M) ->
```

Comment:
I suggest changing the name to `does_module_exist` or `is_module`.

Reply:
Done.